dashboard/app/controllers/reviewable_projects_controller.rb

```diff
@@ -62,12 +62,22 @@ def destroy
   end
 
   def for_level
-    peer_user_ids = current_user.
-      sections_as_student.
-      map(&:followers).
-      flatten.
-      pluck(:student_user_id).
-      select {|student_user_id| current_user.id != student_user_id}
+    peer_user_ids = if DCDO.get('code_review_groups_enabled', false)
+                      current_user.
+                        code_review_groups.
+                        map(&:members).
+                        flatten.
+                        map(&:follower).
+                        pluck(:student_user_id).
+                        select {|student_user_id| current_user.id != student_user_id}
+                    else
+                      current_user.
+                        sections_as_student.
+                        map(&:followers).
```

Comment:
Just for my understanding, why do we map `follower` for code review groups but `followers` for sections? Are those two fields just named differently?

Reply:
In the old logic, the first step (`sections_as_student`) returns an array of `Sections`, which we then get the list of `followers` for each.

In the new logic, `code_review_groups.map(&:members)` returns an array of arrays, each of the inner arrays made up of `CodeReviewGroupMembers`. Those are 1:1 with a follower, so we map to grab the appropriate `follower` entry.